docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+services:
+  api:
+    extends:
+      file: recipes/docker-compose.yml
+      service: api
+    profiles:
+      - 'test'
+    environment:
+      NB_GRAPH_USERNAME: 'DBUSER'
+      NB_GRAPH_ADDRESS: graph
```

Comment:
```suggestion
```
Think you can safely exclude this since it's also hardcoded in the original docker-compose.yml